gulpfile.babel.js

```diff
@@ -82,15 +82,26 @@ const server = browserSync.create();
 // Initialize the BrowserSync server conditionally:
 function serve(done) {
 	if (config.dev.browserSync.live) {
-		server.init({
-			proxy: config.dev.browserSync.proxyURL,
-			port: config.dev.browserSync.bypassPort,
-			liveReload: true
-		});
+		if (config.dev.browserSync.https) {
```

Comment:
If we need to setup different arguments, let's start with the commonalities, e.g.:
`var serverConfig = {
    proxy: config.dev.browserSync.proxyURL,
    liveReload: true
}
if (config.dev.browserSync.https) {
    // Add to serverConfig
} else {
    // Add to serverConfig
}
server.init(serverConfig);
`

Reply:
OK, I'll tackle this next. Figured out it was Prettier extension in VS Code that did all of that formatting change.